### DISCUSSION TITLE
Insert and Select?

### DISCUSSION THREAD

**Discussion by ohashioji** [mikro-orm/mikro-orm#2155] [Q&A] [answered]
Using the query builder, can I insert and return the full row in one query? if so, how?

**Comment by B4nan** [comment, marked as answer]
full column? or you mean full row? depends on the driver, with postgres you have the returning clause, you'd have to use knex to set it tho. i am not aware of other drivers having something similar.

**Reply by ohashioji** [reply]
Fantastic, the knex builder had what I was looking for thanks!
